src/main/java/org/aludratest/testcase/data/TestCaseData.java

```diff
@@ -64,7 +64,7 @@ public TestCaseData(String id, String description, Data[] data) {
     public TestCaseData(String id, String description, Data[] data, boolean ignored) {
         this.id = id;
         this.description = description;
-        this.data = (data == null ? new Data[0] : ArrayUtil.copyOfRange(data, 0, data.length));
+        this.data = ((data == null) ? new Data[0] : ArrayUtil.copyOfRange(data, 0, data.length));
```

Comment:
But brackets around the **whole** expression should be treated superflouous even by Sonar: compare `x = ((a && b) ? c : d)` vs `x = (a && b) ? c : d`